src/Altinn.App.Core/Infrastructure/Clients/Storage/DataClient.cs

```diff
@@ -55,6 +57,18 @@ public DataClient(HttpClient httpClient, IServiceProvider serviceProvider)
         httpClient.DefaultRequestHeaders.Accept.Add(new MediaTypeWithQualityHeaderValue("application/json"));
         httpClient.DefaultRequestHeaders.Accept.Add(new MediaTypeWithQualityHeaderValue("application/xml"));
         _client = httpClient;
+
+        // Configure streaming client
+        var httpClientFactory = serviceProvider.GetRequiredService<IHttpClientFactory>();
+        _streamingClient = httpClientFactory.CreateClient("DataClient.Streaming");
```

Comment:
I'd say this doesn't really have anything to do with stream? Expected timeout should probably be based on payload size and not globally for "streaming requests". One option is to use `CancellationToken`. We can create a `CancellationTokenSource` is the methods where we might get large payloads, set the timeout to something pretty large and then callers can still override that with the passed in `CancellationToken`. What do you think? I think that's a little more to the point as opposed to creating an entire HttpClient for this